### ISSUE TITLE
Need to be DPP

### ISSUE THREAD

**Issue by r-barnes** [r-barnes/faster-unmixer#11]
This isn't DPP
```
def cp_log_ratio_norm(a, b):
    return cp.maximum(a / b, b * cp.inv_pos(a))
```
specifically, `a / b` is not DPP.

@AlexLipp

**Comment by r-barnes** [comment]
From [this doc](https://www.cvxpy.org/tutorial/advanced/index.html#disciplined-parametrized-programming) I see:
> As another example, the quotient expr / p is not DPP-compliant when p is a parameter, but this can be rewritten as expr * p_tilde, where p_tilde is a parameter that represents 1/p.
I'll try to incorporate this wisdom.

**Comment by r-barnes** [comment]
This is fixed in #21
